Assets/Game/Addons/ModSupport/ModManager.cs

```diff
@@ -942,33 +942,43 @@ public static string MakeRelativePath(string path0, string path1)
         /// Seeks asset contributes for the target mod, reading the folder name of each asset.
         /// </summary>
         /// <param name="modInfo">Manifest data for a mod, which will be filled with retrieved contributes.</param>
+        /// <param name="automaticallyRegisterQuestLists">Optional parameter that triggers Quest Lists declaration</param>
         /// <remarks>
         /// Assets are imported from loose files according to folder name,
         /// for example all textures inside `SpellIcons` are considered icon atlases.
         /// This method replicates the same behaviour for mods, doing all the hard work at build time.
         /// Results are stored to json manifest file for performant queries at runtime.
         /// </remarks>
-        public static void SeekModContributes(ModInfo modInfo)
+        public static void SeekModContributes(ModInfo modInfo, bool automaticallyRegisterQuestLists = false)
         {
             List<string> spellIcons = null;
             List<string> booksMapping = null;
+            List<string> questLists = null;
```

Comment:
I think this function should make `modInfo.Contributes.QuestLists` empty if `automaticallyRegisterQuestLists` is false. Right now, you can't disable the feature from the UI. Unchecking the box and resaving does nothing, it's gonna be checked again next time you open it, and the quests will be registered, because they are in the contributes still